cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/mongo/MongoBackendImpl.java

```diff
@@ -17,11 +17,7 @@
  */
```

Comment:
Suggested line for CHANGES_NEXT_RELEASE:

```
[cygnus-commons] [MongoBackend] Proper processing of mongo errors in create index operations (#1756)
```

Reply:
Fixed in [996ac6f](https://github.com/telefonicaid/fiware-cygnus/pull/1760/commits/996ac6fc7d02fa3137637992c4efd2e53447be8c)